util/config.py

```
@@ -9,5 +12,8 @@ def config_from_path(config_path: str, overrides: DictConfig = None) -> DictConf
     for p in path[:-1]:
         env_cfg = env_cfg[p]
     if overrides is not None:
-        env_cfg = OmegaConf.merge(env_cfg, overrides)
+        try:
+            env_cfg = OmegaConf.merge(env_cfg, overrides)
+        except ConfigKeyError:
+            logger.warning(f"Cannot parse overrides when using multienv_mettagrid")
```

Similar to my other review, I expect we'd want to throw here, since `env_cfg` will be in some unknown state afterward.